mypy.ini

```diff
@@ -1,5 +1,35 @@
 [mypy]
-ignore_missing_imports = True
-# required to support namespace packages
-# https://github.com/python/mypy/issues/14057
+# Is the project well-typed?
+strict = False
+
+# Early opt-in even when strict = False
+warn_unused_ignores = True
+warn_redundant_casts = True
+enable_error_code = ignore-without-code
+
+# Support namespace packages per https://github.com/python/mypy/issues/14057
 explicit_package_bases = True
+
+# Disable overload-overlap due to many false-positives
+disable_error_code = overload-overlap
+
+exclude = (?x)(
+    # upstream
+	^build/
+	| ^.tox/
+ 
+ 	# local
+	)
+
+# - distutils._modified has different errors on Python 3.8 [import-untyped], on Python 3.9+ [import-not-found]
```

Comment:
I'm pretty sure distutils concerns are specific to setuptools.

Reply:
True. If a skeleton-based project imports distutils and tests across Python versions, that project can add any necessary config.